tests/integration/test_project.py

```diff
@@ -14,6 +15,14 @@
 FIXTURE_ROOT = os.path.join(os.path.dirname(__file__), 'fixtures')
 
 
+@contextlib.contextmanager
```

Comment:
Your tests are looking good, thanks! I don't mean to be nitpicking, but I don't think you'll be needing this context manager for your tests. As long as your container names start with `lxdock-test-` (correct me if I'm wrong @ellmetha ), cleanup will be automatically taken care of at https://github.com/lxdock/lxdock/blob/bd9d01152e08dc828d8e87de8ef267e35106fd94/lxdock/test/testcases.py#L17